### PR TITLE
24: Create `pull_request_template.md`

### LINKED ISSUE #24: Create PR description template
Typically reminding which details to remember to include, a checklist of testing steps etc

### PR DESCRIPTION
**Pull request by dargaCode**
@angelkbrown here's an example of a pretty "heavy" PR template. Are there any parts of this that you’d find helpful to use in our repos, or ideas it gives you for other things to prompt?

([View the "Rich Diff"](https://github.com/Code-the-Dream-School/ee-prac-team2-front/pull/26/files?short_path=b2496e8#diff-b2496e80299b8c3150b1944450bd81c622e04e13d15c411d291db0927d75fd6b))

====================
This PR resolves Issue #

# Change Summary

- Added component \__a_
- Updated feature \__b_ to do \__c_ when \__d_

## I Had Difficulty With
_I initially tried to implement this using our existing components, but ran into <these issues>, so I decided to add a third party component for this task._

# Code Review

### Starting File

`/src/components/foo.tsx`

## Visual Example
_// Video or screenshot which shows the app with this new code applied_"

## Testing Steps
_// Simple example_
1. Visit `/activities`
2. Observe initial sort is by Name, descending

_// Detailed example_
1. Visit `/activities`
2. Invoke `Sort Popularity ASC` button
   1. Observe Activities sorted by highest Votes first
   2. Observe button text changes to `Sort Popularity DESC`
3. Invoke `Sort Popularity DESC` button
4. Observe Activities sorted by lowest Votes first
5. Observe button text changes to `Sort Popularity ASC`

## Testing Checklist

- [ ] Deleted all debug/log messages
- [ ] Deleted all commented-out code
- [ ] Merged latest `origin/dev` changes into branch
  - [ ] Addressed any merge conflicts
- [ ] Verified no linter errors/warnings
- [ ] Ran app locally
  - [ ] Verified new functionality
  - [ ] Verified no console errors

## Dangers

- This code is at risk of breaking something:
  - _I refactored <this class> to be reused. I tested the other places that class is used as best I could, but other testers and stakeholders should know that there have been code changes in that area._
- This code should not be merged until:
  - _This PR relies on my changes from PR #. Please review and merge that code first, and then I'll update this branch to include those changes before final code review._

## TODO Items

- Breaking this Issue into multiple PRs:
  1. _My code formatter is set to update the entire project based on modern industry standard formatting, so I'm committing the changes it made to improve our code style._
  2. _The second PR will focus on the new functionality specified by the Issue_
- Was unable to implement aspects of this Issue:
  - _Since the purpose of this task was to get something working and shipped quickly, I decided to ignore typescript for now._
  - _I created Issue # to come back and add it later_
  - _Because we are going to revisit this functionality soon, I decided not to spend too much time on <this bit> so that we can prioritize shipping._
    - _I created Issue # to come back and add it later_
- Found a bug or identified other new work:
  - _I realized we didn't include an `Error` component in this milestone. I couldn't find an Issue for it._
    - _I created Issue # so we don't forget to come back and replace this placeholder error_

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+# Change Summary
+
+- Added component \__a_
+- Updated feature \__b_ to do \__c_ when \__d_
+
+## I Had Difficulty With
+
+- \__problem_
+
```
Comment (angelkbrown) [inline .github/pull_request_template.md]: @dargaCode what would you think about keeping just the top two sections just to keep things simple for now?

Comment (dargaCode) [reply .github/pull_request_template.md]: @angelkbrown yeah sounds good! I wanted to create an example with some of the things people might encounter but it's definitely major overkill here =D

**Comment by dargaCode** [comment]
Ok, here's the simplified version! From what I can tell I can't test that it actually populates into new PRs until after it's actually merged.

**Review by angelkbrown — APPROVED** [review]
Looks great! 🎉

**Pull request merged**